### PR TITLE
chore: Fix Attachments and Supplement storage definitions

### PR DESCRIPTION
**Pull request by TheTechmage**
This should fix the issue that @dbluhm mentioned

### CODE REVIEW

File: aries_cloudagent/protocols/present_proof/v2_0/models/pres_exchange.py [hunk 1]
```diff
@@ -334,6 +334,6 @@ class Meta:
     supplements = fields.Nested(
```
Comment (dbluhm) [inline aries_cloudagent/protocols/present_proof/v2_0/models/pres_exchange.py:334]: Need to set `many` to `True`

File: aries_cloudagent/protocols/present_proof/v2_0/models/pres_exchange.py [hunk 2]
```diff
@@ -334,6 +334,6 @@ class Meta:
...
         SupplementSchema(), required=False, description="Supplementary attachment data"
     )
-    attach = fields.Nested(
+    attachments = fields.Nested(
```
Comment (dbluhm) [inline aries_cloudagent/protocols/present_proof/v2_0/models/pres_exchange.py:340]: Same

**Review by dbluhm — CHANGES_REQUESTED** [review]

**Comment by TheTechmage** [comment]
The above changes from Peter were not meant to be a part of this PR. I will work to get those separated into another PR

**Comment by TheTechmage** [comment]
Force push removed the previous two commits. I have moved those to a separate branch, feature/credential-attachments-verifier-tests

**Review by dbluhm — APPROVED** [review]
Looks good, thanks!

**Pull request merged**